WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -7,6 +7,7 @@ open class QuickStartTourGuide: NSObject {
     private var currentTourState: TourState?
     private var suggestionWorkItem: DispatchWorkItem?
     static let notificationElementKey = "QuickStartElementKey"
```

Comment:
What do you think if we move this at the beginning? Just to keep the properties ordered by the accessibility. Like static properties (public/internal/private), stored properties (public/internal/private) and computed properties.